docs/contributing.md

```diff
@@ -24,8 +25,8 @@ For contribution to the website, please refer to the [WebHost README](/WebHostLi
 If you want to contribute to the core, you will be subject to stricter review on your pull requests. It is recommended
 that you get in touch with other core maintainers via the [Discord](https://archipelago.gg/discord).
 
-If you want to add Archipelago support for a new game, please take a look at the [adding games documentation](/docs/adding%20games.md), which details what is required 
-to implement support for a game, as well as tips for how to get started.
+If you want to add Archipelago support for a new game, please take a look at the [adding games documentation](/docs/adding%20games.md),
+which details what is required to implement support for a game, as well as tips for how to get started.
```

Comment:
The comma at least should be removed here, but I think "details [X] as well as tips" is a bit awkward, so this might be better.
It also should either be "tips *on* how to get started" or "tips *for* getting started"
```suggestion
which details what is required to implement support for a game and has tips on how to get started.
```